src/website/layouts/partials/stencilDoc.pug

```diff
@@ -0,0 +1,134 @@
+//mixin usage(usage)
+  // TODO
+
+mixin props(props)
+  if props.length
+    h3 Poperties
+    section.a-table.a-table__options.-bordered.-my--3
+      div
+        table.-text(cellpadding='0', cellspacing='0')
+          thead
+            tr
+              th
+                div Property
+              th
+                div Attribute
+              th
+                div Description
+              th
+                div Type
+              th
+                div Default
+          tbody
+            each prop in props
+              if prop.name !== 'extraClass'
+                tr
+                  td
+                    div
+                      code=prop.name
+                      if prop.required
+                        strong (required)
+                  td
+                    div
+                      if prop.attr
+                        code=prop.attr
+                      else
+                        | --
+                  td
+                    div
+                      if prop.deprecation !== undefined
+                        .-text--danger **[DEPRECATED]** #{prop.deprecation}
+                        br
+                        br
+                      =prop.docs
+                  td
+                    div
+                      code=prop.type
+                  td
+                    div
+                      code=prop.default
```

Comment:
@dani-cl-madrid I think if the default value is not present we should probably ```undefined``` or something descriptive instead of  ``` ```, the later one doesn't look good in the docs. Any thoughts.

Reply:
@srinath-koneru Done! Thanks for your review!